src/main/io/osd.c

```diff
@@ -2537,7 +2537,9 @@ void osdDrawNextElement(void)
     } while(!osdDrawSingleElement(elementIndex) && index != elementIndex);
 
     // Draw artificial horizon last
+#ifndef USE_MSP_DISPLAYPORT
```

Comment:
@ligenxxxx why this is required? I'd like to understand

Reply:
And I think it should be when MSP_DISPLAYPORT is enabled and running (check at runtime), not only compiled into the code.
With current approach, if you compile with `USE_MSP_DISPLAYPORT` AHI stops working even if you do not use DISPLAYPORT

Reply:
This osd elements can make the system less fluid, and I don't know why！

Reply:
Then it should be disabled when MSP_DISPLAYPORT is configured and enabled, not only compiled